conda_build/conda_interface.py

```diff
@@ -4,123 +4,93 @@
 
-import configparser  # NOQA
-configparser = configparser
-
-
-from conda.exports import FileMode, PathType  # NOQA
-FileMode, PathType = FileMode, PathType
-from conda.exports import EntityEncoder  # NOQA
-
-EntityEncoder, FileMode, PathType = EntityEncoder, FileMode, PathType
-
-
-CondaError = try_exports("conda.exceptions", "CondaError")
-CondaHTTPError = try_exports("conda.exceptions", "CondaHTTPError")
-LinkError = try_exports("conda.exceptions", "LinkError")
-LockError = try_exports("conda.exceptions", "LockError")
-NoPackagesFoundError = try_exports("conda.exceptions", "NoPackagesFoundError")
-PaddingError = try_exports("conda.exceptions", "PaddingError")
-UnsatisfiableError = try_exports("conda.exceptions", "UnsatisfiableError")
+from conda.auxlib.packaging import (  # noqa: F401
+    _get_version_from_git_tag as get_version_from_git_tag,
+)
 
-non_x86_linux_machines = try_exports("conda.base.context", "non_x86_linux_machines")
-context = try_exports("conda.base.context", "context")
-context_get_prefix = try_exports("conda.base.context", "get_prefix")
-reset_context = try_exports("conda.base.context", "reset_context")
-get_conda_build_local_url = try_exports("conda.models.channel", "get_conda_build_local_url")
+from conda.exports import TmpDownload, download, handle_proxy_407  # noqa: F401
+from conda.exports import untracked, walk_prefix  # noqa: F401
+from conda.exports import (  # noqa: F401
+    MatchSpec,
+    NoPackagesFound,
+    Resolve,
+    Unsatisfiable,
+    normalized_version,
+)
+from conda.exports import (  # noqa: F401
+    human_bytes,
+    hashsum_file,
+    md5_file,
+    memoized,
+    unix_path_to_win,
+    win_path_to_unix,
+    url_path,
+)
+from conda.exports import get_index  # noqa: F401
+from conda.exports import (  # noqa: F401
+    Completer,
+    InstalledPackages,
+    add_parser_channels,
+    add_parser_prefix,
+    specs_from_args,
+    spec_from_line,
+    specs_from_url,
+)
+from conda.exports import ArgumentParser  # noqa: F401
+from conda.exports import (  # noqa: F401
+    is_linked,
+    linked,
+    linked_data,
+    prefix_placeholder,
+    rm_rf,
+    symlink_conda,
+    package_cache,
+)
+from conda.exports import CondaSession  # noqa: F401
+from conda.exports import StringIO, input, lchmod, TemporaryDirectory  # noqa: F401
+from conda.exports import VersionOrder  # noqa: F401
+
+from conda.core.package_cache import ProgressiveFetchExtract  # noqa: F401
+from conda.models.dist import Dist, IndexRecord  # noqa: F401
+
+import configparser  # noqa: F401
+
+from conda.exports import FileMode, PathType  # noqa: F401
+from conda.exports import EntityEncoder  # noqa: F401
+from conda.exceptions import (  # noqa: F401
+    CondaError,
+    CondaHTTPError,
+    LinkError,
+    LockError,
+    NoPackagesFoundError,
+    PaddingError,
+    UnsatisfiableError,
+)
+from conda.base.context import (  # noqa: F401
+    non_x86_linux_machines,
+    context,
+    get_prefix,
```

Comment:
we need this to be called differently:

```suggestion
    get_prefix as context_get_prefix,
```